services/web-client/source/class/qxapp/layout/LayoutManager.js

```diff
@@ -21,15 +21,15 @@ qx.Class.define("qxapp.layout.LayoutManager", {
     this._pane = new qx.ui.splitpane.Pane("horizontal");
 
     const settingsWidth = 500;
```

Comment:
Perhaps  ``settingsWidth`` and similar parameters could be defined as qx properties with defaults so it can be modified from outside ??